vignettes/howto-dm-rows.Rmd

```diff
@@ -18,25 +18,25 @@ source("setup/setup.R")
 
 
 This tutorial introduces the methods {dm} provides for modifying the data in the tables of a relational model.
-There are 6 methods:
+There are 5 methods:
 
 * [`dm_rows_insert()`](#insert) - adds new rows
```

Comment:
`dm_rows_append()`

Reply:
Should all instances of `dm_rows_insert()` be replaced by `dm_rows_append()`? 
Or should the vignette mention both of these options?

Reply:
We have both, but `dm_rows_insert()` seems to be much less useful than `dm_rows_append()` .

Reply:
They are different, though -- `dm_rows_insert()` will ditch duplicates or give an error, in some cases this is what we want. We can take this on in a separate PR though.